js/src/admin/components/MailPage.js

```diff
@@ -121,11 +122,29 @@ export default class MailPage extends Page {
                 ],
               })}
 
-            {Button.component({
-              type: 'submit',
-              className: 'Button Button--primary',
-              children: app.translator.trans('core.admin.email.submit_button'),
-              disabled: !this.changed(),
+            {FieldSet.component({
```

Comment:
Nitpick: Could we just use `<FieldSet></FieldSet>` here and get rid a bunch of braces ?

Reply:
Fixed throughout, thanks!